### PR TITLE
Pre-assessment accessibility reports Data Ajax, Data tables, Data-JSO…

### PR DESCRIPTION
**Pull request by ipaksc**
…N, Equal height, Footnotes, and Form validation

### CODE REVIEW

File: src/plugins/data-ajax/reports/pre-a11y-1.json [hunk 1]
```diff
@@ -0,0 +1,589 @@
...
+    "@type": ["earl:Assertion", "acr:AssessmentReport"],
+
+    "earl:subject": {
+        "@id": "_:subject",
+        "dct:references": "https://wet-boew.github.io/gcweb-compiled-demos/wetboew-demos/data-ajax/data-ajax-en.html",
+        "@type": [
+            "earl:TestSubject",
+            "schema:WebPage"
+        ],
+        "dct:description": "Accessibility conformance reports of Data Ajax",
```
Comment (duboisp) [inline src/plugins/data-ajax/reports/pre-a11y-1.json:13]:
```suggestion
        "dct:description": "Data ajax plugin state in basic HTML mode and when it has imported content in the page. The imported content are out of scope, only the data-ajax wrapper is.",
```
This is a description of the test subject and not of the whole pre-assessment report.

File: src/plugins/data-ajax/reports/pre-a11y-1.json [hunk 2]
```diff
@@ -0,0 +1,589 @@
...
+        "earl:pointer": {
+            "@type": "oa:CssSelector",
+            "@value": "data-ajax-after,data-ajax-append,data-ajax-before,data-ajax-prepend,data-ajax-replace,#filter-id,filter-selector"
```
Comment (duboisp) [inline src/plugins/data-ajax/reports/pre-a11y-1.json:16]: This property must be a valid CSS selector as specified by the type [oa:CssSelector](https://www.w3.org/TR/annotation-model/#css-selector)

Here I think you wanted to select some data attribute and class.
```suggestion
            "@value": "[data-ajax-after],[data-ajax-append],[data-ajax-before],[data-ajax-prepend],[data-ajax-replace],#filter-id,.filter-selector"
```
The modified CSS selector that I suggest do select 15 nodes in the test subject.

File: src/plugins/data-ajax/reports/pre-a11y-1.json [hunk 3]
```diff
@@ -0,0 +1,589 @@
...
+        "earl:mainAssertor": {
+            "foaf:name": "Ilya Pak (Github: @ipaksc)",
+            "foaf:homepage": "https://github.com/ipaksc",
+            "@type": ["earl:Assertor", "foaf:Person"]
+        }
+    },
+
+    "dct:date": "2023-06-30",
+
+    "dct:description": "Evaluation of all WCAG 2.1 SC at level AA.",
```
Comment (duboisp) [inline src/plugins/data-ajax/reports/pre-a11y-1.json:33]:
```suggestion
    "dct:description": "Pre-evaluation of all WCAG 2.1 SC at level AA.",
```

File: src/plugins/equalheight/reports/a11y-1-en.hbs [hunk 4]
```diff
@@ -0,0 +1,15 @@
+---
+{
+	"title": "Accessibility assessment #1 - Equal height",
+	"language": "en",
+	"description": "Partial. Evaluation of new Success Criteria at Level A and Level AA of WCAG 2.1 and WCAG 2.2",
```
Comment (duboisp) [inline src/plugins/equalheight/reports/a11y-1-en.hbs:5]:
```suggestion
	"title": "Accessibility pre-assessment #1 - Equal height",
	"language": "en",
	"description": "Partial. Pre-evaluation of Success Criteria at Level A and Level AA of WCAG 2.1",
```

File: src/plugins/equalheight/reports/a11y-1-fr.hbs [hunk 5]
```diff
@@ -0,0 +1,15 @@
+---
+{
+	"title": "Assessment d'accessibilité #1 - Égalisation des hauteurs",
+	"language": "fr",
+	"description": "Partielle. Évaluation des nouveaux critères de succès aux niveaux A et AA de WCAG 2.1 et WCAG 2.2",
```
Comment (duboisp) [inline src/plugins/equalheight/reports/a11y-1-fr.hbs:5]:
```suggestion
	"title": "Pré-assessment de l'accessibilité #1 - Égalisation des hauteurs",
	"language": "fr",
	"description": "Partielle. Pré-évaluation des critères de succès aux niveaux A et AA du WCAG 2.1",
```

File: src/plugins/equalheight/reports/pre-a11y-1.json [hunk 6]
```diff
@@ -0,0 +1,589 @@
...
+    "@type": ["earl:Assertion", "acr:AssessmentReport"],
+
+    "earl:subject": {
+        "@id": "_:subject",
+        "dct:references": "https://wet-boew.github.io/gcweb-compiled-demos/wetboew-demos/equalheight/equalheight-en.html",
+        "@type": [
+            "earl:TestSubject",
+            "schema:WebPage"
+        ],
+        "dct:description": "Accessibility conformance reports of Equal height",
```
Comment (duboisp) [inline src/plugins/equalheight/reports/pre-a11y-1.json:13]: Please adjust this test subject description similarly to my change proposal for the data-ajax pre-assessment.

File: src/plugins/equalheight/reports/pre-a11y-1.json [hunk 7]
```diff
@@ -0,0 +1,589 @@
...
+        "earl:pointer": {
+            "@type": "oa:CssSelector",
+            "@value": "gc-features"
```
Comment (duboisp) [inline src/plugins/equalheight/reports/pre-a11y-1.json:16]: I don't think you means to select the custom element `<gc-features>` here. Probably the css selector is `.wb-eqht`

File: src/plugins/equalheight/reports/pre-a11y-1.json [hunk 8]
```diff
@@ -0,0 +1,589 @@
...
+        "earl:mainAssertor": {
+            "foaf:name": "Ilya Pak (Github: @ipaksc)",
+            "foaf:homepage": "https://github.com/ipaksc",
+            "@type": ["earl:Assertor", "foaf:Person"]
+        }
+    },
+
+    "dct:date": "2023-06-30",
+
+    "dct:description": "Evaluation of all WCAG 2.1 SC at level AA.",
```
Comment (duboisp) [inline src/plugins/equalheight/reports/pre-a11y-1.json:33]:
```suggestion
    "dct:description": "Pre-evaluation of all WCAG 2.1 SC at level AA.",
```

File: src/plugins/footnotes/reports/a11y-1-en.hbs [hunk 9]
```diff
@@ -0,0 +1,15 @@
+---
+{
+	"title": "Accessibility assessment #1 - Footnotes",
+	"language": "en",
+	"description": "Partial. Evaluation of new Success Criteria at Level A and Level AA of WCAG 2.1 and WCAG 2.2",
```
Comment (duboisp) [inline src/plugins/footnotes/reports/a11y-1-en.hbs:5]:
```suggestion
	"title": "Accessibility pre-assessment #1 - Footnotes",
	"language": "en",
	"description": "Partial. Pre-evaluation of Success Criteria at Level A and Level AA of WCAG 2.1",
```

File: src/plugins/footnotes/reports/a11y-1-fr.hbs [hunk 10]
```diff
@@ -0,0 +1,15 @@
+---
+{
+	"title": "Assessment d'accessibilité #1 - Notes de bas de page",
+	"language": "fr",
+	"description": "Partielle. Évaluation des nouveaux critères de succès aux niveaux A et AA de WCAG 2.1 et WCAG 2.2",
```
Comment (duboisp) [inline src/plugins/footnotes/reports/a11y-1-fr.hbs:5]:
```suggestion
	"title": "Pré-assessment de l'accessibilité #1 - Notes de bas de page",
	"language": "fr",
	"description": "Partielle. Pré-évaluation des critères de succès aux niveaux A et AA de WCAG 2.1",
```

File: src/plugins/footnotes/reports/pre-a11y-1.json [hunk 11]
```diff
@@ -0,0 +1,589 @@
...
+    "@type": ["earl:Assertion", "acr:AssessmentReport"],
+
+    "earl:subject": {
+        "@id": "_:subject",
+        "dct:references": "https://wet-boew.github.io/gcweb-compiled-demos/wetboew-demos/footnotes/footnotes-en.html",
+        "@type": [
+            "earl:TestSubject",
+            "schema:WebPage"
+        ],
+        "dct:description": "Accessibility conformance reports of Footnotes",
```
Comment (duboisp) [inline src/plugins/footnotes/reports/pre-a11y-1.json:13]: Review the description which describe the scope of the test subject and not the description of this report.

File: src/plugins/footnotes/reports/pre-a11y-1.json [hunk 12]
```diff
@@ -0,0 +1,589 @@
...
+        "earl:pointer": {
+            "@type": "oa:CssSelector",
+            "@value": "gc-features"
```
Comment (duboisp) [inline src/plugins/footnotes/reports/pre-a11y-1.json:16]: This CSS selector are not correct

File: src/plugins/footnotes/reports/pre-a11y-1.json [hunk 13]
```diff
@@ -0,0 +1,589 @@
...
+        "earl:mainAssertor": {
+            "foaf:name": "Ilya Pak (Github: @ipaksc)",
+            "foaf:homepage": "https://github.com/ipaksc",
+            "@type": ["earl:Assertor", "foaf:Person"]
+        }
+    },
+
+    "dct:date": "2023-06-30",
+
+    "dct:description": "Evaluation of all WCAG 2.1 SC at level AA.",
```
Comment (duboisp) [inline src/plugins/footnotes/reports/pre-a11y-1.json:33]:
```suggestion
    "dct:description": "Pre-evaluation of all WCAG 2.1 SC at level AA.",
```

File: src/plugins/formvalid/reports/a11y-1-en.hbs [hunk 14]
```diff
@@ -0,0 +1,15 @@
+---
+{
+	"title": "Accessibility assessment #1 - Form validation",
+	"language": "en",
+	"description": "Partial. Evaluation of new Success Criteria at Level A and Level AA of WCAG 2.1 and WCAG 2.2",
```
Comment (duboisp) [inline src/plugins/formvalid/reports/a11y-1-en.hbs:5]: Make the same edit as proposed in the other pages

File: src/plugins/formvalid/reports/a11y-1-fr.hbs [hunk 15]
```diff
@@ -0,0 +1,15 @@
+---
+{
+	"title": "Assessment d'accessibilité #1 - Validation de formulaires",
+	"language": "fr",
+	"description": "Partielle. Évaluation des nouveaux critères de succès aux niveaux A et AA de WCAG 2.1 et WCAG 2.2",
```
Comment (duboisp) [inline src/plugins/formvalid/reports/a11y-1-fr.hbs:5]: Make the same edit as proposed in the other pages

File: src/plugins/formvalid/reports/pre-a11y-1.json [hunk 16]
```diff
@@ -0,0 +1,589 @@
...
+    "@type": ["earl:Assertion", "acr:AssessmentReport"],
+
+    "earl:subject": {
+        "@id": "_:subject",
+        "dct:references": "https://wet-boew.github.io/gcweb-compiled-demos/wetboew-demos/formvalid/formvalid-en.html",
+        "@type": [
+            "earl:TestSubject",
+            "schema:WebPage"
+        ],
+        "dct:description": "Accessibility conformance reports of Form validation",
```
Comment (duboisp) [inline src/plugins/formvalid/reports/pre-a11y-1.json:13]: Make the same edit as proposed in the other pages

File: src/plugins/formvalid/reports/pre-a11y-1.json [hunk 17]
```diff
@@ -0,0 +1,589 @@
...
+        "earl:pointer": {
+            "@type": "oa:CssSelector",
+            "@value": "gc-features"
```
Comment (duboisp) [inline src/plugins/formvalid/reports/pre-a11y-1.json:16]: Make the same edit as proposed in the other pages

File: src/plugins/formvalid/reports/pre-a11y-1.json [hunk 18]
```diff
@@ -0,0 +1,589 @@
...
+        "earl:mainAssertor": {
+            "foaf:name": "Ilya Pak (Github: @ipaksc)",
+            "foaf:homepage": "https://github.com/ipaksc",
+            "@type": ["earl:Assertor", "foaf:Person"]
+        }
+    },
+
+    "dct:date": "2023-06-30",
+
+    "dct:description": "Evaluation of all WCAG 2.1 SC at level AA.",
```
Comment (duboisp) [inline src/plugins/formvalid/reports/pre-a11y-1.json:33]: Make the same edit as proposed in the other pages

File: src/plugins/tables/reports/a11y-1-en.hbs [hunk 19]
```diff
@@ -0,0 +1,15 @@
+---
+{
+	"title": "Accessibility assessment #1 - Tables",
+	"language": "en",
+	"description": "Partial. Evaluation of new Success Criteria at Level A and Level AA of WCAG 2.1 and WCAG 2.2",
```
Comment (duboisp) [inline src/plugins/tables/reports/a11y-1-en.hbs:5]: Make the same edit as proposed in the other pages

File: src/plugins/tables/reports/a11y-1-fr.hbs [hunk 20]
```diff
@@ -0,0 +1,15 @@
+---
+{
+	"title": "Assessment d'accessibilité #1 - Tables",
+	"language": "fr",
+	"description": "Partielle. Évaluation des nouveaux critères de succès aux niveaux A et AA de WCAG 2.1 et WCAG 2.2",
```
Comment (duboisp) [inline src/plugins/tables/reports/a11y-1-fr.hbs:5]: Make the same edit as proposed in the other pages

File: src/plugins/tables/reports/pre-a11y-1.json [hunk 21]
```diff
@@ -0,0 +1,589 @@
...
+    "@type": ["earl:Assertion", "acr:AssessmentReport"],
+
+    "earl:subject": {
+        "@id": "_:subject",
+        "dct:references": "https://wet-boew.github.io/gcweb-compiled-demos/wetboew-demos/tables/tables-en.html",
+        "@type": [
+            "earl:TestSubject",
+            "schema:WebPage"
+        ],
+        "dct:description": "Accessibility conformance reports of Tables",
```
Comment (duboisp) [inline src/plugins/tables/reports/pre-a11y-1.json:13]: Make the same edit as proposed in the other pages

File: src/plugins/tables/reports/pre-a11y-1.json [hunk 22]
```diff
@@ -0,0 +1,589 @@
...
+        "earl:pointer": {
+            "@type": "oa:CssSelector",
+            "@value": "wb-tables"
```
Comment (duboisp) [inline src/plugins/tables/reports/pre-a11y-1.json:16]: I think you means the CSS class selector

```suggestion
            "@value": ".wb-tables"
```

File: src/plugins/tables/reports/pre-a11y-1.json [hunk 23]
```diff
@@ -0,0 +1,589 @@
...
+        "earl:mainAssertor": {
+            "foaf:name": "Ilya Pak (Github: @ipaksc)",
+            "foaf:homepage": "https://github.com/ipaksc",
+            "@type": ["earl:Assertor", "foaf:Person"]
+        }
+    },
+
+    "dct:date": "2023-06-30",
+
+    "dct:description": "Evaluation of all WCAG 2.1 SC at level AA.",
```
Comment (duboisp) [inline src/plugins/tables/reports/pre-a11y-1.json:33]: Make the same edit as proposed in the other pages

File: src/plugins/wb-data-json/reports/a11y-1-en.hbs [hunk 24]
```diff
@@ -0,0 +1,15 @@
+---
+{
+	"title": "Accessibility assessment #1 - Data JSON",
+	"language": "en",
+	"description": "Partial. Evaluation of new Success Criteria at Level A and Level AA of WCAG 2.1 and WCAG 2.2",
+	"tag": "data-json",
```
Comment (duboisp) [inline src/plugins/wb-data-json/reports/a11y-1-en.hbs:6]: Make the same edit as proposed in the other pages

File: src/plugins/wb-data-json/reports/a11y-1-fr.hbs [hunk 25]
```diff
@@ -0,0 +1,15 @@
+---
+{
+	"title": "Assessment d'accessibilité #1 - Data JSON",
+	"language": "fr",
+	"description": "Partielle. Évaluation des nouveaux critères de succès aux niveaux A et AA de WCAG 2.1 et WCAG 2.2",
+	"tag": "data-json",
```
Comment (duboisp) [inline src/plugins/wb-data-json/reports/a11y-1-fr.hbs:6]: Make the same edit as proposed in the other pages

File: src/plugins/wb-data-json/reports/pre-a11y-1.json [hunk 26]
```diff
@@ -0,0 +1,589 @@
...
+    "@type": ["earl:Assertion", "acr:AssessmentReport"],
+
+    "earl:subject": {
+        "@id": "_:subject",
+        "dct:references": "https://wet-boew.github.io/gcweb-compiled-demos/wetboew-demos/wb-data-json/data-json-en.html",
+        "@type": [
+            "earl:TestSubject",
+            "schema:WebPage"
+        ],
+        "dct:description": "Accessibility conformance reports of Data JSON",
```
Comment (duboisp) [inline src/plugins/wb-data-json/reports/pre-a11y-1.json:13]: Make the same edit as proposed in the other pages

File: src/plugins/wb-data-json/reports/pre-a11y-1.json [hunk 27]
```diff
@@ -0,0 +1,589 @@
...
+        "earl:pointer": {
+            "@type": "oa:CssSelector",
+            "@value": "data-json-replacewith,data-json-after,data-wb-json,data-json-append,data-json-before,data-json-prepend,data-json-replace,data-json-replacewith"
```
Comment (duboisp) [inline src/plugins/wb-data-json/reports/pre-a11y-1.json:16]: Please adjust to make it a valid CSS selector which we can use as a parameter with the JS function `document.querySelectorAll( "" )` to find all evaluated HTML elements

File: src/plugins/wb-data-json/reports/pre-a11y-1.json [hunk 28]
```diff
@@ -0,0 +1,589 @@
...
+        "earl:mainAssertor": {
+            "foaf:name": "Ilya Pak (Github: @ipaksc)",
+            "foaf:homepage": "https://github.com/ipaksc",
+            "@type": ["earl:Assertor", "foaf:Person"]
+        }
+    },
+
+    "dct:date": "2023-06-30",
+
+    "dct:description": "Evaluation of all WCAG 2.1 SC at level AA.",
```
Comment (duboisp) [inline src/plugins/wb-data-json/reports/pre-a11y-1.json:33]: Make the same edit as proposed in the other pages

**Review by duboisp — CHANGES_REQUESTED** [review]
Great work on this

A few note:
* [ ] Need to add an empty line at the end of each file.
* [ ] Improve the page title
* [ ] Little content change in the report
* [ ] Replicate the similar change in the other report.
* [ ] Add references to the report from the matching documentation page
   * [ ] data-ajax
   * [ ] equalheight
   * [ ] footnotes
   * [ ] form validation
   * [ ] data table
   * [ ] data-json

For adding the reference (link) to the report, you will need to modify the documentation page associated to the plugin. You can find those documentation page here: https://github.com/wet-boew/wet-boew/tree/master/site/pages/docs/ref/data-ajax
You simply need to add them near by the links to the working example, a HTML code block like:
```html
<div lang="en">
<h2>Evaluation and report</h2>
<ul>
	<li>Accessibility pre-assessment #1
		<ul>
			<li><a href="../../../demos/add-cal/reports/1-assessment-wcag21-en.html" hreflang="en">English report</a></li>
			<li><a href="../../../demos/add-cal/reports/1-assessment-wcag21-fr.html" hreflang="fr">French report</a></li>
		</ul>
	</li>
</ul>
</div>
```
Similar to: https://github.com/wet-boew/wet-boew/blob/master/site/pages/docs/ref/add-cal/add-cal-en.hbs#L30-L44 and apply the same change to the corresponding French documentation page.

File: src/plugins/data-ajax/reports/a11y-1-en.hbs [hunk 29]
```diff
@@ -0,0 +1,15 @@
...
+	"tag": "data-ajax",
+	"parentdir": "data-ajax",
+	"parentPage": "Data Ajax",
+	"parentPageURL": "data-ajax",
+	"altLangPage": "a11y-1-fr.html",
+	"dateModified": "2023-07-04",
+	"layout": "assessment_wrote_en-en.hbs",
+	"reportURL": "pre-a11y-1.json"
+}
+---
```
[the remark this answers is not in the document]
Comment (ipaksc) [reply src/plugins/data-ajax/reports/a11y-1-en.hbs]: done

File: src/plugins/data-ajax/reports/a11y-1-fr.hbs [hunk 30]
```diff
@@ -0,0 +1,15 @@
...
+	"tag": "data-ajax",
+	"parentdir": "data-ajax",
+	"parentPage": "Data Ajax",
+	"parentPageURL": "data-ajax",
+	"altLangPage": "a11y-1-en.html",
+	"dateModified": "2023-07-04",
+	"layout": "assessment_wrote_en-fr.hbs",
+	"reportURL": "pre-a11y-1.json"
+}
+---
```
[the remark this answers is not in the document]
Comment (ipaksc) [reply src/plugins/data-ajax/reports/a11y-1-fr.hbs]: done

File: src/plugins/data-ajax/reports/a11y-1-en.hbs [hunk 31]
```diff
@@ -0,0 +1,15 @@
...
+{
+	"title": "Accessibility assessment #1 - Data Ajax",
```
[the remark this answers is not in the document]
Comment (ipaksc) [reply src/plugins/data-ajax/reports/a11y-1-en.hbs]: done

File: src/plugins/data-ajax/reports/a11y-1-fr.hbs [hunk 32]
```diff
@@ -0,0 +1,15 @@
...
+{
+	"title": "Assessment d'accessibilité #1 - Data Ajax",
```
[the remark this answers is not in the document]
Comment (ipaksc) [reply src/plugins/data-ajax/reports/a11y-1-fr.hbs]: done

File: src/plugins/data-ajax/reports/pre-a11y-1.json (hunk 2 above)
Comment (ipaksc) [reply src/plugins/data-ajax/reports/pre-a11y-1.json]: done

File: src/plugins/data-ajax/reports/pre-a11y-1.json (hunk 3 above)
Comment (ipaksc) [reply src/plugins/data-ajax/reports/pre-a11y-1.json]: done

File: src/plugins/equalheight/reports/a11y-1-en.hbs (hunk 4 above)
Comment (ipaksc) [reply src/plugins/equalheight/reports/a11y-1-en.hbs]: done

File: src/plugins/equalheight/reports/a11y-1-fr.hbs (hunk 5 above)
Comment (ipaksc) [reply src/plugins/equalheight/reports/a11y-1-fr.hbs]: done

File: src/plugins/equalheight/reports/pre-a11y-1.json (hunk 6 above)
Comment (ipaksc) [reply src/plugins/equalheight/reports/pre-a11y-1.json]: done

File: src/plugins/equalheight/reports/pre-a11y-1.json (hunk 7 above)
Comment (ipaksc) [reply src/plugins/equalheight/reports/pre-a11y-1.json]: fixed

File: src/plugins/equalheight/reports/pre-a11y-1.json (hunk 8 above)
Comment (ipaksc) [reply src/plugins/equalheight/reports/pre-a11y-1.json]: done

File: src/plugins/wb-data-json/reports/pre-a11y-1.json (hunk 27 above)
Comment (ipaksc) [reply src/plugins/wb-data-json/reports/pre-a11y-1.json]: done

File: src/plugins/wb-data-json/reports/pre-a11y-1.json (hunk 26 above)
Comment (ipaksc) [reply src/plugins/wb-data-json/reports/pre-a11y-1.json]: done

File: src/plugins/wb-data-json/reports/a11y-1-fr.hbs (hunk 25 above)
Comment (ipaksc) [reply src/plugins/wb-data-json/reports/a11y-1-fr.hbs]: done

File: src/plugins/tables/reports/a11y-1-en.hbs (hunk 19 above)
Comment (ipaksc) [reply src/plugins/tables/reports/a11y-1-en.hbs]: done

File: src/plugins/formvalid/reports/pre-a11y-1.json (hunk 18 above)
Comment (ipaksc) [reply src/plugins/formvalid/reports/pre-a11y-1.json]: done

File: src/plugins/formvalid/reports/pre-a11y-1.json (hunk 17 above)
Comment (ipaksc) [reply src/plugins/formvalid/reports/pre-a11y-1.json]: done

File: src/plugins/formvalid/reports/pre-a11y-1.json (hunk 16 above)
Comment (ipaksc) [reply src/plugins/formvalid/reports/pre-a11y-1.json]: done

File: src/plugins/formvalid/reports/a11y-1-fr.hbs (hunk 15 above)
Comment (ipaksc) [reply src/plugins/formvalid/reports/a11y-1-fr.hbs]: done

File: src/plugins/formvalid/reports/a11y-1-en.hbs (hunk 14 above)
Comment (ipaksc) [reply src/plugins/formvalid/reports/a11y-1-en.hbs]: done

File: src/plugins/footnotes/reports/pre-a11y-1.json (hunk 13 above)
Comment (ipaksc) [reply src/plugins/footnotes/reports/pre-a11y-1.json]: done

File: src/plugins/footnotes/reports/pre-a11y-1.json (hunk 12 above)
Comment (ipaksc) [reply src/plugins/footnotes/reports/pre-a11y-1.json]: fixed

File: src/plugins/wb-data-json/reports/a11y-1-en.hbs (hunk 24 above)
Comment (ipaksc) [reply src/plugins/wb-data-json/reports/a11y-1-en.hbs]: done

File: src/plugins/data-ajax/reports/a11y-1-fr.hbs [hunk 33]
```diff
@@ -0,0 +1,15 @@
...
+	"language": "fr",
+	"description": "Partielle. Évaluation des nouveaux critères de succès aux niveaux A et AA de WCAG 2.1 et WCAG 2.2",
```
[the remark this answers is not in the document]
Comment (ipaksc) [reply src/plugins/data-ajax/reports/a11y-1-fr.hbs]: fixed

File: src/plugins/footnotes/reports/a11y-1-en.hbs (hunk 9 above)
Comment (ipaksc) [reply src/plugins/footnotes/reports/a11y-1-en.hbs]: fixed

File: src/plugins/tables/reports/pre-a11y-1.json (hunk 23 above)
Comment (ipaksc) [reply src/plugins/tables/reports/pre-a11y-1.json]: done

File: src/plugins/tables/reports/pre-a11y-1.json (hunk 21 above)
Comment (ipaksc) [reply src/plugins/tables/reports/pre-a11y-1.json]: done

**Comment by ipaksc** [comment]
@duboisp Hi Pierre, how do I fix this error. 

* At dist/demos/tables/reports/a11y-1-en.html:98:
internally linking to ../../../demos/tables/table-en.html, which does not exist

File: src/plugins/tables/reports/a11y-1-en.hbs [hunk 34]
```diff
@@ -0,0 +1,15 @@
+---
+{
+	"title": "Accessibility pre-assessment #1 - Tables",
+	"language": "en",
+	"description": "Partial. Pre-evaluation of Success Criteria at Level A and Level AA of WCAG 2.1",
+	"tag": "tables",
+	"parentdir": "tables",
+	"parentPage": "Tables",
+	"parentPageURL": "table",
```
Comment (duboisp) [inline src/plugins/tables/reports/a11y-1-en.hbs:9]: The parent page filename do include a "s" before the language suffix. see https://github.com/wet-boew/wet-boew/blob/master/src/plugins/tables/tables-en.hbs 

```suggestion
	"parentPageURL": "tables",
```

File: src/plugins/wb-data-json/reports/a11y-1-en.hbs [hunk 35]
```diff
@@ -0,0 +1,15 @@
+---
+{
+	"title": "Accessibility pre-assessment #1 - Data JSON",
+	"language": "en",
+	"description": "Partial. Pre-evaluation of Success Criteria at Level A and Level AA of WCAG 2.1",
+	"tag": "data-json",
+	"parentdir": "wb-data-json",
+	"parentPage": "Data JSON",
+	"parentPageURL": "wb-data-json",
```
Comment (duboisp) [inline src/plugins/wb-data-json/reports/a11y-1-en.hbs:9]:
```suggestion
	"parentPageURL": "data-json",
```

File: src/plugins/wb-data-json/reports/a11y-1-fr.hbs [hunk 36]
```diff
@@ -0,0 +1,15 @@
+---
+{
+	"title": "Pré-assessment de l'accessibilité #1 - Data JSON",
+	"language": "fr",
+	"description": "Partielle. Pré-évaluation des critères de succès aux niveaux A et AA du WCAG 2.1",
+	"tag": "data-json",
+	"parentdir": "wb-data-json",
+	"parentPage": "Data JSON",
+	"parentPageURL": "wb-data-json",
```
Comment (duboisp) [inline src/plugins/wb-data-json/reports/a11y-1-fr.hbs:9]:
```suggestion
	"parentPageURL": "data-json",
```

**Review by duboisp — CHANGES_REQUESTED** [review]
> @duboisp Hi Pierre, how do I fix this error.
> 
>     * At dist/demos/tables/reports/a11y-1-en.html:98:
>       internally linking to ../../../demos/tables/table-en.html, which does not exist

That issue is coming from the breadcrumb, in your .hbs file inside the reports folder. See my request change which should fix that issue.

File: src/plugins/tables/reports/a11y-1-en.hbs (hunk 34 above)
Comment (ipaksc) [reply src/plugins/tables/reports/a11y-1-en.hbs]: fixed

File: src/plugins/wb-data-json/reports/a11y-1-en.hbs (hunk 35 above)
Comment (ipaksc) [reply src/plugins/wb-data-json/reports/a11y-1-en.hbs]: fixed

File: src/plugins/wb-data-json/reports/a11y-1-fr.hbs (hunk 36 above)
Comment (ipaksc) [reply src/plugins/wb-data-json/reports/a11y-1-fr.hbs]: fixed

**Review by duboisp — APPROVED** [review]

**Pull request merged**